etc/bibtex.profile

```diff
@@ -0,0 +1,13 @@
+# Firejail profile for bibtex
+# This file is overwritten after every install/update
+# Persistent local customizations
+include bibtex.local
+# Persistent global definitions
+# added by caller profile
+#include globals.local
```

Comment:
caller profile --> uncomment this line

Reply:
And remove the ```# added by caller profile``` line I presume?

Reply:
@florianbegusch Yes that's correct. As there is no exception now, the comment becomes useless and we always remove it to avoid confusion.

Reply:
I'm not quite getting it.  
IMHO I should:
* remove ```#include globals.local``` from all profiles except for **latex-common.profile**
* in **latex-common.profile** I should remove ```# added by caller profile``` and uncomment ```#include globals.local```

Reply:
Oh I get it, you are accounting for the case that latex-common.profile gets altered or removed

Reply:
@florianbegusch Indeed. That's the 'hidden' assumption in this context. To ease profile management we use this kind of common file to avoid having to make changes later in all relevant profiles. So when a future upstream change in tex causes us to change something we only need to do it in the one common file. Takes some time to get used to that workflow, but it pays of later.

Reply:
> @florianbegusch Indeed. That's the 'hidden' assumption in this context. To ease profile management we use this kind of common file to avoid having to make changes later in all relevant profiles. So when a future upstream change in tex causes us to change something we only need to do it in the one common file. Takes some time to get used to that workflow, but it pays of later.

Thanks for the insight